lib/mavenlink_api.php

```diff
@@ -222,6 +222,34 @@ function deleteWorkspaceExpense($workspaceId, $expenseId)
   {
     return $this->deleteModel('Expense', $workspaceId, $expenseId);
   }
+  
+    // Field Sets
```

Comment:
We probably don't need these comments. The function names seem to tell the story nicely enough.